docs/src/content/docs/zh-cn/guides/components.mdx

```diff
@@ -194,3 +194,49 @@ import { Icon } from '@astrojs/starlight/components';
 下面显示了所有可用图标的列表及其关联的名称。点击图标以复制其组件代码。
 
 <IconsList />
+
+### Code
```

Comment:
It seems that all the previous component headings have been translated, I guess it would make sense to translate this one too, for consistency. Any thoughts?

Reply:
en.. I'll think of a suitable translation.